eris.go

```diff
@@ -126,18 +131,15 @@ func Cause(err error) error {
 
 // StackFrames returns the trace of an error in the form of a program counter slice.
 // Use this method if you want to pass the eris stack trace to some other error tracing library.
-func StackFrames(e error) []uintptr {
-	if e == nil {
-		return []uintptr{}
-	}
-	switch err := e.(type) {
-	case *wrapError:
-		return err.StackFrames()
-	case *rootError:
-		return err.StackFrames()
-	default:
-		return []uintptr{}
+func StackFrames(err error) []uintptr {
+	for err != nil {
+		switch err := err.(type) {
+		case *rootError:
+			return *err.stack
```

Comment:
Just to confirm the logic, since the rootErr stack now contains the frames of wrapped error in correct order, we are skipping the wrapErr case.

Reply:
yep, exactly